controllers/cloudstackmachine_controller_utils.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022.
```

Comment:
The machine controller file was getting bloated.

Broke some of the methods out here.

Reply:
I'm not sure I agree with splitting out methods for the same object across multiple files. That'll make it more confusing to navigate and test, I feel, especially for a new maintainer. I'm inclined to keep all the relevant code in a single file instead of hiding some functions

Reply:
I personally don't consider module size (in lines of code) to be particularly sinful.  The current 317 lines plus the 77 from this module don't concern me.

The fact that RemoveManagedAffinity is only called from reconcileDelete() in the machine controller also dissuades me from breaking it out into a utility module.  (I'd usually put shared functions there).

I don't see any calls to ResolveManagedAffinity.

Reply:
Cool, removed the resolve method and moved all back into the machine controller.